security-demo/release/app-agent-conf/agent/agent.conf

```diff
@@ -0,0 +1,39 @@
+agent {
+    bind_address = "192.168.56.102"
+    bind_port = "9088"
+    data_dir = "./.app-data"
+    log_level = "DEBUG"
+    plugin_dir = "./app-agent-conf/agent/plugin"
+    server_address = "192.168.56.102"
+    server_port = "8081"
+    socket_path ="/tmp/app-agent.sock"
+    trust_bundle_path = "./app-agent-conf/agent/dummy_root_ca.crt"
+    trust_domain = "example.org"
+    umask = ""
+    join_token = ""
```

Comment:
What's the meaning of join_token? Can we remove this field in config file?

Reply:
Yes, we can remove these unused configs.